data/assets/examples/renderable/renderablepointcloud/advanced/pointcloud_multitextured.asset

```diff
@@ -0,0 +1,94 @@
+-- Example of point clouds where multiple textures are used for the points,
+-- based on information in the dataset. The dataset may be either CSV or Speck format.
+
+-- Load data from a CSV file. This requires additional information to be provided through
+-- the DataMapping:
+-- 1) Which column in the dataset that corresponds to the texture, and
+-- 2) a separate file that maps that value to a texture file
+local Node = {
+  Identifier = "RenderablePointCloud_Example_MultiTextured_CSV",
+  Renderable = {
+    Type = "RenderablePointCloud",
+    File = asset.resource("../data/multitextured_csv/textured_points.csv"),
+    DataMapping = {
+      -- The name of the column in the CSV file that corresponds to the texture (should
+      -- be an integer)
+      TextureColumn = "texture",
+      -- A Texture mapping file that provides information about which value/index
+      -- corresponds to which texture file
+      TextureMapFile = asset.resource("../data/multitextured_csv/texturemap.tmap")
+    },
+    Texture = {
+      -- Where to find the texture files (in this case, in the OpenSpace data folder)
+      Folder = openspace.absPath("${DATA}")
+    },
+    UseAdditiveBlending = false
+  },
+  GUI = {
+    Name = "Multi-Textured Points",
+    Path = "/Examples"
+  }
+}
+
+
+-- Interpolated
```

Comment:
Maybe split this asset into three separate assets?

Reply:
Do we really want even more files? 

In my mind, this asset really shows just one thing, which is how to add multi-textured points (and that they also work with interpolated points and speck files).  Any user who tries to do this should be considered an advanced user and maybe then it does not matter if there are multiple scene graph nodes per file. 

But don't have a super-strong opinion, just thought it would be easier to navigate one file instead of three.  If people generally think it's better to split them up, I'm ok with that

Reply:
I guess if we don't want them to appear in the documentation page that makes sense. The splitting was for the docs page I presume